mesheryctl/internal/cli/root/registry/update.go

```diff
@@ -53,9 +54,19 @@ mesheryctl registry update --spreadsheet-id [id] --spreadsheet-cred [base64 enco
 mesheryctl registry update --spreadsheet-id 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw --spreadsheet-cred $CRED
 // Updating models in the meshery/meshery repo based on flag
 mesheryctl registry update --spreadsheet-id 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw --spreadsheet-cred $CRED --model "[model-name]"
+
+// Updating models from a local CSV directory
+mesheryctl registry update --csv-dir /path/to/csv-directory
+
+// Example to update with both Google Sheets and CSV, CSV takes precedence
+mesheryctl registry update --spreadsheet-id [id] --spreadsheet-cred [base64 encoded spreadsheet credential] --csv-dir [/path/to/csv-directory]
 	`,
 	PreRunE: func(cmd *cobra.Command, args []string) error {
 
+		if csvDir == "" && (spreadsheeetID == "" || spreadsheeetCred == "") {
+			return errors.New("please provide a CSV directory or both spreadsheet-id and spreadsheet-cred")
```

Comment:
Can you follow same behavior  being used in #13152 
```suggestion
			return errors.New("please provide a CSV directory or both spreadsheet-id and spreadsheet-cred")
```